### ISSUE TITLE
"NotStartedError: not started" when trying to close orbitdb.

### ISSUE THREAD

**Issue by vinkabuki** [orbitdb/orbit-db#947]
Hello, I have an issue with closing orbitdb - it hangs (not always and only for one instance) or throws error and hangs (always for more than one instance). I've created a minimal repo, so you can see it yourself. It doesn't throw an error if you don't initialize keyvaluestore - something with ipfs pubsub I guess, but maybe there is something wrong with my setup, I was trying to close and even drop those kvstores, but it didn't work.

Here is demo repo: https://github.com/vinkabuki/closeorbitdb
Also why-is-node-running logs:  https://justpaste.it/8dk53

The error message is:

```
(node:24923) UnhandledPromiseRejectionWarning: NotStartedError: not started
    at Function.use (/closeorbitdb/node_modules/ipfs-core/cjs/src/utils/service.js:91:13)
    at Service.use (/closeorbitdb/node_modules/ipfs-core/cjs/src/utils/service.js:104:26)
    at peers (/closeorbitdb/node_modules/ipfs-core/cjs/src/components/pubsub.js:44:36)
    at Object.peers (/closeorbitdb/node_modules/ipfs-core-utils/cjs/src/with-timeout-option.js:19:14)
    at checkPeers (/closeorbitdb/node_modules/ipfs-pubsub-1on1/src/wait-for-peers.js:5:37)
    at Timeout._onTimeout (/closeorbitdb/node_modules/ipfs-pubsub-1on1/src/wait-for-peers.js:17:19)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)

**Comment by haadcode** [comment]
I think you’ve found a bug here 👍

It seems to me that the [interval](https://github.com/ipfs-shipyard/ipfs-pubsub-1on1/blob/master/src/wait-for-peers.js#L15) doesn’t get canceled upon [closing](https://github.com/ipfs-shipyard/ipfs-pubsub-1on1/blob/master/src/direct-channel.js#L64) the channel so the polling for peers happens even after OrbitDB/IPFS has been stopped. Thus the error.

We’d need to cancel the interval (‘clearInterval(interval)’) upon closing the channel. Would be happy to review and merge a PR that does that.

**Comment by vinkabuki** [comment]
@haadcode I've created a PR

**Comment by tabcat** [comment]
oops this isnt quit closed yet, my bad.

**Comment by haadcode** [comment]
See https://github.com/ipfs-shipyard/ipfs-pubsub-1on1/pull/29#issuecomment-1017264576.

@vinkabuki can you confirm that the fix published in ^ fixes the problem for you?

**Comment by vinkabuki** [comment]
@haadcode Yes, the problem disappeared. thanks

**Comment by tabcat** [comment]
fixed in version 0.28.1
